src/lib/components/Grid/README.mdx

```diff
@@ -53,14 +53,30 @@ See [API](#api) for all available options.
   markup** like GridItem or Cell is necessary for your items. But it's there
   when you really need it—see [Advanced Layouts](#advanced-layouts).
 
+- For forms, use rather the [FormLayout](/components/form-layout) component
+  which is designed specifically for that purpose.
+
 - The Grid component is so powerful that it enables you to build even very
   advanced layouts **without** having to write **a single line of custom CSS.**
   See [Advanced Layouts](#advanced-layouts) for more.
 
-👉 The default layout has one column, auto-sized rows and gaps of
-[size 4](/foundation/spacing). Defaults for all Grid API options can be
+👉 Vertical margin of items inside Grid is reset to zero.
+
+## Stack
```

Comment:
I would consider moving the contents of the `Stack` section to the `Basic Usage` section. I think usage with no props is the basic usage.